docs/src/main/sphinx/connector/prometheus.md

```diff
@@ -52,11 +52,13 @@ The following configuration properties are available:
   - Where to find Prometheus coordinator host.
   - `http://localhost:9090`
 * - `prometheus.query.chunk.size.duration`
-  - The duration of each query to Prometheus.
+  - The duration of each query to Prometheus. 
+    This can also be modified via the `query_chunk_size_duration` session property.
   - `1d`
 * - `prometheus.max.query.range.duration`
   - Width of overall query to Prometheus, will be divided into
-    `prometheus.query.chunk.size.duration` queries.
+    `prometheus.query.chunk.size.duration` queries. 
+    This can also be modified via the `max_query_range_duration` session property.
```

Comment:
The equivalent catalog session property is `max_query_range_duration`.

Reply:
Done